fs/reader/reader.go

```diff
@@ -268,10 +266,10 @@ func WithReader(sr *io.SectionReader) CacheOption {
 	}
 }
 
-func digestVerifier(id uint32, chunkDigestStr string) (digest.Verifier, error) {
-	chunkDigest, err := digest.Parse(chunkDigestStr)
+func digestVerifier(id uint32, digestStr string) (digest.Verifier, error) {
+	digest, err := digest.Parse(digestStr)
 	if err != nil {
-		return nil, errors.Wrap(err, "invalid chunk: no digset is recorded")
+		return nil, errors.Wrap(err, "no digset is recorded")
```

Comment:
nit: maybe "invalid blob: no digest is recorded", but up to you

Reply:
This func is not used. It's part of [verifier](https://github.com/awslabs/soci-snapshotter/blob/main/fs/reader/reader.go#L135) which should be removed completely but it's out of scope of this PR.